packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -42,6 +43,8 @@ import ChangePasswordConfirmation from './cfd-change-password-confirmation';
 import TradingPlatformIcon from '../Assets/svgs/trading-platform';
 import { observer, useStore } from '@deriv/stores';
 import { useCfdStore } from '../Stores/Modules/CFD/Helpers/useCfdStores';
+import { useActiveWallet, useFeatureFlags } from '@deriv/hooks';
+import { useInvalidateQuery } from '@deriv/api';
```

Comment:
I believe we should configure prettier and let it handle the import orders.

Reply:
@arshad-rao-deriv it would be super helpful! In this case would be great to configure ESlint too, but I'm not sure it's the scope of this card :)

Reply:
Yea I agree with @sergei-deriv about this. Plus I don't know how to 😂 . Maybe bringing it up to Ako or someone 🤔

Reply:
The auto ordering is not in the scope of this card but let's order the files manually till then